src/api/java/com/minecolonies/api/entity/ai/pathfinding/AbstractWalkToProxy.java

```diff
@@ -295,7 +295,12 @@ public void addToProxyList(final BlockPos pos)
      */
     public boolean isLivingAtSiteWithMove(final EntityLiving entity, final int x, final int y, final int z, final int range)
     {
-        return EntityUtils.isLivingAtSiteWithMove(entity, x, y, z, range);
+        if(!EntityUtils.isLivingAtSiteWithMove(entity, x, y, z, range))
+        {
+            EntityUtils.tryMoveLivingToXYZ(entity, x, y, z);
+            return false;
+        }
+        return true;
```

Comment:
Nothing, I just noted that if someone wants to know why you changed something there